llvm/test/MC/AArch64/armv9.6a-lsui.s

```diff
@@ -0,0 +1,395 @@
+# NOTE: Assertions have been autogenerated by utils/update_mca_test_checks.py
```

Comment:
Is this comment accurate? `llvm-mca` is a different tool to `llvm-mc`, and there's a `update_mc_test_checks.py` script which I'd expect to be used instead.

Generally, these tests are hard to read when the input and expected output are far apart. Maybe `update_mc_test_checks.py` will help with that?

Reply:
Done